From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@seanfreiburg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tfitzmac commented:
I have incorporated this change with other changes in a different pull request.

rmca14 commented:
Thanks to all. Closing this PR due to tfitzmac's comment.

tankungx committed:
Azure Data Lake Storage draft doc

tankungx committed:
Azure Data Lake Storage draft doc

tankungx committed:
Azure Data Lake Storage draft doc

parakhj committed:
Adding AAD Article Template

parakhj committed:
Adding GitHub idp instructions

parakhj committed:
Updating AAD Documentation

parakhj committed:
Minor update

parakhj committed:
Updating URL

parakhj committed:
Adding private preview message

parakhj committed:
Adding lowercase note

parakhj committed:
Adding TOC

tankungx committed:
update ADLS doc

deepakjangid-MSFT committed:
Update CSharp ServiceFabric docs for Redhat

rapatchi committed:
Update service-fabric-get-started-linux.md

Adding redhat support for java and yo

rapatchi committed:
Update service-fabric-cli.md

Adding installation steps for RHEL

rapatchi committed:
Update service-fabric-create-your-first-linux-application-with-java.md

Change to make the doc consistent for redhat

ggailey777 committed:
Create an A-B test for the Favorite Functions section removal

ggailey777 committed:
Acrolinx tweaks and fixed the Login back to MVC standard language

dominicbetts committed:
Rename article

dominicbetts committed:
Initial version of device model articles

dominicbetts committed:
Fix Acrolinx issues

dominicbetts committed:
Add TOC entries

ShawnJackson committed:
edit pass: app-service-plan-manage

ShawnJackson committed:
edit pass: app-service-plan-manage

ShawnJackson committed:
Update app-service-plan-manage.md

rapatchi committed:
Update service-fabric-create-your-first-linux-application-with-csharp.md

Addressing review comments 
Removing subsection of migration for project.json to .csproj

rapatchi committed:
Update service-fabric-create-your-first-linux-application-with-java.md

Addressed review comments

rapatchi committed:
Update service-fabric-get-started-linux.md

Addressed review comments

rapatchi committed:
Update service-fabric-cli.md

Addressed review comments

rapatchi committed:
Merge branch 'master' into service_fabric_redhat_csharp

dominicbetts committed:
Address feedback

Commit:
Add instructions to install SF on RHEL

GitHubber17 committed:
edit pass: author-visually

GitHubber17 committed:
edit pass: adjust headings

ecfan committed:
Add notes about how some triggers use long polling

ecfan committed:
Revised to address PM feedback

rapatchi committed:
Update service-fabric-cli.md

Adding sudo for some commands

ggailey777 committed:
Merge branch 'master' of github.com:MicrosoftDocs/azure-docs-pr into experiment

ggailey777 committed:
Roll back the control to the current version

ggailey777 committed:
Tom feedback

tankungx committed:
Update ADLS doc.

tankungx committed:
resolve conflict

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into connector-billing

ecfan committed:
Tweaks

Commit:
Add Jupyter-based quick start for emotion API

Commit:
Merged latest master

Commit:
Add Jupyter-based quick start for emotion API

Commit:
Make "Ubuntu" and "Red Hat Enterprise 7.4" bold

ecfan committed:
Edits per PM feedback

tankungx committed:
update ADLS doc

christiankuhtz committed:
typo

kadamajay56 committed:
(ForumsTeam)repeated "named"word

There was a repeated word "named" under How to create the NSG from the front end section so i have removed the excessive word.

Sogup committed:
Update azure-backup-limits.md

carolinacmoravia committed:
M22851: Clarification required

Hello, @davidmu1,

Translator has reported possible source content issue.  

Please, help to check my comment tag added into the article and reply with explanation if source text should be fixed, or it is by design. If fix is needed, please, let me know either if you would like me to fix it in this PR, or if you will fix it in another PR. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

ecfan committed:
Tweaks

itechedit committed:
edit pass: active-directory-saas-bamboo-hr-tutorial

itechedit committed:
removing file

carolinacmoravia committed:
C11990: LOCBLOCKER: Unloc string due to extra ```

Hello, @tamram, 

Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

itechedit committed:
Update active-directory-saas-bamboo-hr-tutorial.md

itechedit committed:
Update active-directory-saas-bamboo-hr-tutorial.md

itechedit committed:
Update active-directory-saas-bamboo-hr-tutorial.md

Commit:
Fix obsolete statement about Video API preview

Brenduns committed:
Updates for subscriptions

pareshmu committed:
ER location-provider update

sanjeev3 committed:
Create media-services-content-moderation.md

ecfan committed:
Updated per feedback and refresh content + screenshots

sanjeev3 committed:
Update media-services-portal-analyze.md

Brenduns committed:
revisions

sanjeev3 committed:
Add files via upload

sanjeev3 committed:
Update TOC.md

Brenduns committed:
revisions

sanjeev3 committed:
Update media-services-content-moderation.md

ecfan committed:
Fix broken link

sanjeev3 committed:
Update media-services-content-moderation.md

ecfan committed:
Fix Acrolinx issues

sanjeev3 committed:
Update media-services-portal-analyze.md

sanjeev3 committed:
Update media-services-portal-analyze.md

ecfan committed:
Fix Acrolinx issues

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into connector-billing

PRMerger13 committed:
Merge pull request #32886 from christiankuhtz/patch-52

typo

Commit:
deladpetime

roygara committed:
Java fixes.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into JQU

roygara committed:
Metadata update.

Commit:
Merge pull request #32957 from MarkusVi/deladpetime

deladpetime

roygara committed:
Acrolinx feedback.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into JQU

Commit:
deladpetime02

Commit:
tocfix02

MikeDodaro committed:
Fix typo: (Couldn't find web results-> )

saurabhsensharma committed:
Update backup-azure-backup-import-export.md

Edit Offline backup flow

ktoliver committed:
Merge pull request #31836 from ggailey777/experiment

Experiment: Create an A-B test for the Favorite Functions section removal

roygara committed:
Upload links

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into JQU

roygara committed:
Fixing link

ktoliver committed:
Merge pull request #32960 from MarkusVi/deladpetime02

deladpetime02

spelluru committed:
Chrome and Edge only

ShawnJackson committed:
Update active-directory-saas-bamboo-hr-tutorial.md

Lisaco88 committed:
Merge pull request #32535 from ecfan/connector-billing

Add notes about how some triggers use long polling

PRMerger18 committed:
Merge pull request #32964 from MikeDodaro/FixCodeTypoWebSrchSDK

Fix typo: (Couldn't find web results-> )

Lisaco88 committed:
Merge pull request #32961 from MarkusVi/tocfix02

tocfix02

parakhj committed:
Removing github

PRMerger17 committed:
Merge pull request #32958 from roygara/JQU

Updates to Java QS

spelluru committed:
Edge and Chrome only

parakhj committed:
Adding github docs

PRMerger15 committed:
Merge pull request #32946 from pareshmu/patch-5

ER location-provider update

parakhj committed:
Updating twitter preview tag

ShawnJackson committed:
Merge pull request #32929 from itechedit/active-directory-saas-bamboo-hr-tutorial

edit pass: active-directory-saas-bamboo-hr-tutorial

parakhj committed:
Updating articles

ktoliver committed:
Merge pull request #32963 from saurabhsensharma/patch-14

Update backup-azure-backup-import-export.md

parakhj committed:
Updating whitespace

sethmanheim committed:
Changing get started to quick starts per fundamentals feedback

ShawnJackson committed:
Update author-visually.md

ShawnJackson committed:
Merge pull request #32517 from GitHubber17/vsts-1172602

edit pass: author-visually

kevinlam1 committed:
Update logic-apps-limits-and-config.md

Throughput limits incorrectly indicated that limits applied to a logic apps instance instead of a logic apps resource.

kevinlam1 committed:
Update logic-apps-limits-and-config.md

Fixing Acrolynx warnings

PRMerger13 committed:
Merge pull request #32977 from kevinlam1/patch-16

Update logic-apps-limits-and-config.md

rjagiewich committed:
Merge pull request #32231 from dominicbetts/suite-device-model-format

Add article describing device model JSON format

rjagiewich committed:
Merge pull request #32970 from parakhj/Github

GitHub idP

PRMerger17 committed:
Merge pull request #31483 from deepakjangid-MSFT/service_fabric_redhat_csharp

Update CSharp ServiceFabric docs for Redhat

rjagiewich committed:
Merge pull request #32512 from yihuaz/sf_rhel_doc_yihzhang

Add instructions to install/remove SF on RHEL

rjagiewich committed:
Merge pull request #32950 from sanjeev3/master

Added Content Moderator content to Media Services Docs

sanjeev3 committed:
Update media-services-content-moderation.md

sanjeev3 committed:
Update media-services-content-moderation.md

Commit:
Merge pull request #32891 from kadamajay56/patch-3

(ForumsTeam)repeated "named" word

PRMerger16 committed:
Merge pull request #32981 from sanjeev3/master

Added the Next steps Section.

sethmanheim committed:
Fixing TOC

Commit:
Merge pull request #32968 from spelluru/browser0206

Chrome and Edge only

spelluru committed:
Edge and Chrome

PRMerger10 committed:
Merge pull request #32983 from spelluru/browser0206-2

Edge and Chrome only

PRMerger17 committed:
Merge pull request #32984 from spelluru/browser0206-3

Edge and Chrome

tankungx committed:
update adls doc

rayne-wiselman committed:
update freshness for A2A articles

tankungx committed:
update adls doc,update storage explorer doc.

PRMerger19 committed:
Merge pull request #32986 from rayne-wiselman/rayne-february7

update freshness for A2A articles

rayne-wiselman committed:
updating freshness dates

rayne-wiselman committed:
updating freshness

linda33wj committed:
Clarify VPC support

linda33wj committed:
Clarify Xero connector supported tables

linda33wj committed:
Fix S3 connector broken link

PRMerger20 committed:
Merge pull request #32990 from rayne-wiselman/rayne-february7-3

updating freshness

linda33wj committed:
Fix AKV permission and add all data store support

linda33wj committed:
Refine ADLS SP & MSI permission requirements

linda33wj committed:
Clarify incremental copy prepare steps

PRMerger19 committed:
Merge pull request #32992 from linda33wj/master

Update ADF content

linda33wj committed:
Update all connector topics to add cred in AKV support

rayne-wiselman committed:
removing folders that were created in error in master branch

rayne-wiselman committed:
putting one folder back because too many files for PR

rayne-wiselman committed:
remove folder added to master by mistake

Sogup committed:
Update azure-backup-limits.md

Sogup committed:
Update azure-backup-limits.md

PRMerger10 committed:
Merge pull request #32935 from carolinacmoravia/patch-608

C11990: LOCBLOCKER: Unloc string due to extra ```

PRMerger17 committed:
Merge pull request #32912 from carolinacmoravia/patch-605

M22851: Clarification required

asmalser-msft committed:
Minor changes for attribute customization

Commit:
Merge pull request #32998 from asmalser-msft/asana-pingboard

Minor changes for attribute customization

Commit:
updates for freshness

PRMerger15 committed:
Merge pull request #33002 from Blackmist/freshness

updates for freshness

Brenduns committed:
Update azure-stack-subscribe-plan-provision-vm.md

Fixing a missing formatting mark.

ktoliver committed:
Merge pull request #32971 from sethmanheim/toc-fix

Changing get started to quick starts per fundamentals feedback

ktoliver committed:
Merge pull request #32995 from rayne-wiselman/rayne-february7-5

removing folders that were created in error in master branch

Lisaco88 committed:
Added word space in first line

Lisaco88 committed:
Deleted accidental asterisk in metadata

ktoliver committed:
Merge pull request #32996 from rayne-wiselman/rayne-february7-6

remove folder added to master by mistake

Lisaco88 committed:
Merge pull request #28531 from tankungx/ADLSASE

Azure Data Lake Storage draft doc

Lisaco88 committed:
Merge pull request #32944 from Brenduns/subscriptions

Updates for subscriptions

rjagiewich committed:
Merge pull request #32988 from rayne-wiselman/rayne-february7-2

updating freshness dates

Lisaco88 committed:
Merge pull request #32994 from linda33wj/master

Update all ADF connector topics to add cred in AKV support

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #33003 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #32873 from drdarshan/master

Add Jupyter notebooks based quickstart for emotion APIs.

Ja-Dunn committed:
Merge pull request #32279 from ShawnJackson/app-service-plan-manage

edit pass: app-service-plan-manage

v-shils committed:
Merge pull request #32902 from Sogup/patch-34

Update azure-backup-limits.md
